engines/pyfunc-ensembler-job/Dockerfile

```diff
@@ -1,8 +1,13 @@
 FROM apache/spark-py:v3.1.3
 
+# Switch to user root so we can add additional jars and configuration files.
+USER root
```

Comment:
q: won't it possible using the user 0?

Reply:
Ah oops my bad, didn't notice there was user 0 line below ><" I removed the `USER 0` line instead so as to make this file look similar to the batch predictor Dockerfile in Merlin: https://github.com/caraml-dev/merlin/blob/9e78611f3343cda6fa7f82b72b96051d065214ad/python/batch-predictor/docker/base.Dockerfile#L18